util.py

```diff
@@ -294,11 +294,15 @@ def get_lan_address_without_netifaces():
     # Get the local ip address by creating a socket for a (random) internet ip
     :return: the local ip address
     """
-    s = socket(AF_INET, SOCK_DGRAM)
-    s.connect(("192.0.2.0", 80)) # TEST-NET-1, guaranteed to not be connected => no callbacks
-    local_ip = s.getsockname()[0]
-    s.close()
-    return local_ip
+    try:
+        s = socket(AF_INET, SOCK_DGRAM)
+        s.connect(("192.0.2.0", 80)) # TEST-NET-1, guaranteed to not be connected => no callbacks
+        local_ip = s.getsockname()[0]
+        s.close()
+        return local_ip
+    except Exception as exception:
```

Comment:
General exception, should only catch what can be thrown. I believe it throws a socket.gaierror if not being able to connect.